### PR TITLE
00-move-notifications-to-ui

### PR DESCRIPTION
**Pull request by kevinmamaqi**

### CODE REVIEW

File: packages/ui/src/components/molecules/Notifications/Notification.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import { FlexBox, colors, device, dimensions, font } from '../../../styles'
+import { Icon, Text } from '../../atoms'
+import { NActions, TNotification, TVariant } from './types'
+import { NotificationsContext } from './context'
+import { useNotifications } from './useNotifications'
+
+const ContainerStyled = styled(FlexBox)<{ variant: TVariant }>`
+  flex-direction: row;
+  justify-content: space-between;
+  align-self: end;
```
Comment (NataliaR-BCN) [inline packages/ui/src/components/molecules/Notifications/Notification.tsx:12]: Should be props

Comment (marmorrei) [inline packages/ui/src/components/molecules/Notifications/Notification.tsx:12]: flex-direction and justify-content should go as attributes

File: packages/ui/src/components/molecules/Notifications/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  @media only ${device.Laptop} {
+    width: 25vw;
+    margin-top: ${dimensions.spacing.base};
+    margin-bottom: ${dimensions.spacing.base};
+  }
+`
+
+const ContentStyled = styled(FlexBox)`
+  align-items: start;
```
Comment (marmorrei) [inline packages/ui/src/components/molecules/Notifications/Notification.tsx:29]: Should go as attribute

File: packages/ui/src/components/molecules/Notifications/Notification.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      <IconStyled
+        name="close"
+        $wght={700}
+        onClick={() => removeNotification(id)}
+      />
+    </ContainerStyled>
+  )
+}
+
+export default Notification
```
Comment (marmorrei) [inline packages/ui/src/components/molecules/Notifications/Notification.tsx:86]: Change to named export

File: packages/ui/src/components/molecules/Notifications/Notifications.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  return hasNotifications ? (
+    <NotificationsContainer data-testid="notifications-test">
+      {Object.values(state.byId).map((notification) => (
+        <Notification key={notification.id} {...notification} />
+      ))}
+    </NotificationsContainer>
+  ) : null
+}
+
+export default Notifications
```
Comment (marmorrei) [inline packages/ui/src/components/molecules/Notifications/Notifications.tsx:27]: Change to named export

**Review by marmorrei — APPROVED** [review]

**Review by NataliaR-BCN — APPROVED** [review]

**Pull request merged**